lib/hexpm/web/controllers/package_controller.ex

```diff
@@ -65,12 +65,13 @@ defmodule Hexpm.Web.PackageController do
 
   defp package(conn, package, releases, release, type) do
     release = Releases.preload(release)
+    latest_release_with_docs = Enum.find(releases, & &1.has_docs)
 
     docs_assigns =
       cond do
-        type == :package and Enum.any?(releases, &(&1.has_docs)) ->
+        type == :package and latest_release_with_docs ->
```

Comment:
If there is possibility for `nil` we usually do not use the boolean operators, instead use `&&`.